src/list-sorting/index.js

```diff
@@ -1,3 +1,15 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+  let haystackLength = haystack.length;
+
+  for (let j = haystackLength - 1; j >= 0; j--) {
+    let index = haystack[j];
+
+    if (typeof index != "object") return haystack.lastIndexOf(needle);
```

Comment:
Embrace braces. Prefer `Array.isArray` when checking if an object is an array.

Reply:
> Embrace braces. Prefer `Array.isArray` when checking if an object is an array.

Thanks... will look into that.